index.js

```diff
@@ -87,11 +87,17 @@ module.exports = function (schema, options) {
     var mainUpdateMethod = mongooseMajorVersion < 5 ? 'update' : 'updateMany';
     var mainUpdateWithDeletedMethod = mainUpdateMethod + 'WithDeleted';
 
-    function updateDocumentsByQuery(schema, conditions, updateQuery, callback) {
+    function updateDocumentsByQuery(schema, conditions, updateQuery, options, callback) {
+        if (typeof options === 'function') {
+            callback = options;
+            options = { multi: true };
+        } else if (!options) {
+            options = { multi: true };
+        }
```

Comment:
🤔 So I'm wondering about how we handle `options` and whether we always need to set `{ multi: true }` on `options`.

If `{ multi: true }` is always required then maybe we should change this slightly
```suggestion
            callback = options;
            options = { multi: true };
        } else {
            options = { 
              ...options,
              multi: true,
            };
        }
```

Reply:
🤔 Hang on, looking at what was there before we're returning early because we don't want to do anything else in this function. Previously we were setting options to `{ multi: true }` because none of the other params in `options` mattered. 

Now we're returning everything in options and the `{multi: true}` but is it the case that we really only want to include the `session` in the options object we pass down here?

Reply:
I think you're right, we do need to include the `session` in options here. It was possible getting unset